modules/installation-user-infra-machines-advanced.adoc

```diff
@@ -135,36 +135,39 @@ kind: MachineConfig
 metadata:
   labels:
     machineconfiguration.openshift.io/role: worker
-  name: 98-var-log-partition
+  name: 98-var-partition
 spec:
   config:
     ignition:
       version: 3.1.0
     storage:
       disks:
-      - device: /dev/nvme0n1
+      - device: /dev/<device_name> <1>
         wipeTable: false
         partitions:
-        - sizeMiB: 47000
-          startMiB: 47000
-          label: var-log
+        - sizeMiB: <partition_size>
+          startMiB: <partition_start_size> <2>
+          label: var
       filesystems:
-        - path: /var/log
-          device: /dev/disk/by-partlabel/var-log
+        - path: /var
+          device: /dev/disk/by-partlabel/var
           format: xfs
     systemd:
       units:
-        - name: var-log.mount
+        - name: var.mount
           enabled: true
           contents: |
             [Unit]
             Before=local-fs.target
             [Mount]
-            Where=/var/log
-            What=/dev/disk/by-partlabel/var-log
+            Where=/var
+            What=/dev/disk/by-partlabel/var
             [Install]
             WantedBy=local-fs.target
 ----
++
+<1> The storage device name of the disk that you want to partition.
+<2> A minimum of 25000 mebibytes is recommended. To fill data partition space, `startMiB` sets the size of the root filesystem. If no value is specified, or if the specified value is too small, future reinstalls might overwrite the beginning of the data partition.
```

Comment:
This isn't very clear.  How about:

"A minimum value of 25000 mebibytes is recommended. The root filesystem is automatically resized to fill all available space up to the specified offset. If no value is specified or the specified value is smaller than the recommended minimum, the resulting root filesystem will be too small, and future reinstalls of {op-system} might overwrite the beginning of the data partition."

Reply:
Updated